zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/SystemContext.java

```diff
@@ -110,12 +114,32 @@ private void validateConfiguration() {
 
     validateExperimentalConfigs(cluster, brokerCfg.getExperimental());
 
+    validateExporters(brokerCfg.getExporters());
+
     final var security = brokerCfg.getNetwork().getSecurity();
     if (security.isEnabled()) {
       validateNetworkSecurityConfig(security);
     }
   }
 
+  private void validateExporters(final Map<String, ExporterCfg> exporters) {
+    final Set<Entry<String, ExporterCfg>> entries = exporters.entrySet();
+    final Stream<Entry<String, ExporterCfg>> badExporters =
+        entries.stream()
+            .filter(
+                entry ->
+                    entry.getValue().getClassName() == null
+                        || entry.getValue().getJarPath() == null);
+    final Optional<Entry<String, ExporterCfg>> optExporter = badExporters.findFirst();
+
+    if (optExporter.isPresent()) {
+      final Entry<String, ExporterCfg> exporter =
+          optExporter.get(); // TODO: change to get more exporters
+      throw new IllegalArgumentException(
+          "Couldn't find a valid exporter for " + exporter.getKey() + "!");
+    }
```

Comment:
:x: It would be good to get all misconfigured exporters in the error message. Please remove the TODO as well once it is done.

:wrench: The error message can be more descriptive - "Expected to find a 'className' or 'jarPath" configured for the exproter. But couldn't find a valid one for the following exporters []".

Reply:
Thanks for the suggestions!